Horion/Command/Commands/EnchantCommand.cpp

```diff
@@ -13,15 +48,31 @@ EnchantCommand::~EnchantCommand()
 
 bool EnchantCommand::execute(std::vector<std::string>* args)
 {
-	assertTrue(args->size() > 1);
+	assertTrue(args->size() > 1); // this line stops the command from been run if the proper args are not given
+
 	int enchantId = 0;
 	int enchantLevel = 32767;
-	if (args->size() > 2)
-		enchantLevel = assertInt(args->at(2));
 
-	if (strcmp(args->at(1).c_str(),"all") != 0)
+	try {
+		// convert string to back to lower case
+		std::string data = args->at(1);
+		std::for_each(data.begin(), data.end(), [](char& c) {
+			c = ::tolower(c);
+		});
```

Comment:
`std::transform(data.begin(), data.end(),  data.begin(), ::tolower);`

Reply:
changed 4 line code to 2 line code